Bot/Commands/DND/RaceInfo.cs

```diff
@@ -20,6 +21,9 @@ public class RaceInfo : ISlashCommand
 
         public async Task CommandEvent(SocketSlashCommand slashCommand)
         {
+            bool tos = await CheckDNDTos(slashCommand);
```

Comment:
This makes the code repetitive if we have to include this line in the CommandEvent() method of every command we make, move it to the interaction handler, you still have access to the SocketSlashCommand there